main.py

```
@@ -0,0 +1,99 @@
+            # Send answer
+            answer = result.get("answer", "No answer available")
+            yield {"data": json.dumps({"type": "answer", "text": answer})}
+
+            # If successful query, also send data
+            if result.get("success", False) and result.get("data"):
+                yield {"data": json.dumps({"type": "data", "text": str(result["data"])})}
+
+            yield {"data": json.dumps({"type": "done"})}
+
+        return EventSourceResponse(event_generator())
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)